### DISCUSSION TITLE
برنامه هایی که Subscription پشتیبانی میکنند

### DISCUSSION THREAD

**Discussion by SaintShit** [Gozargah/Marzban#61] [Show and tell]
برنامه هایی که Subscription ساپورت میکنند:

 - Nekoray _( Windows, Linux, Mac )_
 - Clash For Windows _( Windows, Linux, Mac )_
 - V2rayN _( Windows )_
 - V2rayU _( Mac )_
 -  Clash For Android _( Android )_
 - V2RayNG _( Android )_
 - Shadowrocket _( IOS )_
 - Stash _( IOS )_
 - NapsternetV _( IOS )_
 - FairVPN _( IOS )_

اگر برنامه دیگه ای میشناسید که خوب کار میکنه به همین پیام ریپلای بدین که اضافه کنیم به لیست.

**Comment by ghost** [comment]
- [NekoBox](https://github.com/MatsuriDayo/NekoBoxForAndroid) ( _Android_ )
- [Matsuri](https://github.com/MatsuriDayo/Matsuri) ( _Android_ ) **deprecated**
- [SagerNet](https://github.com/SagerNet/SagerNet) ( _Android_ )
- [Clash Meta for Android](https://github.com/MetaCubeX/ClashMetaForAndroid) ( _Android_ )
- [FoXray](https://apps.apple.com/app/foxray/id6448898396) (_IOS_)

**Reply by sammlahy916-droid** [reply]
سلام

**Comment by mansour-ghasemi** [comment]
NapsternetV (ios) does not support subscription!

**Reply by Ramintn** [reply]
The android version supports are you sure about ios?

**Reply by peymansh68** [reply]
yes

**Comment by M03ED** [comment]
‎Wings X ( iOS )

**Comment by hrprogrammer** [comment]
V2box(ios)

**Comment by bijandna** [comment]
ARMod VPN (Android)

**Reply by Ramintn** [reply]
This app doesnt support subscription links or at least i couldn't find

**Comment by abedin70** [comment]
هیدیفای عالیه
خودش هر چند ساعت ابدیت میگیره اتومات از سابسکربشن
و امکان انتخاب هوشمندهم داره ک هر کدوم پینگ بالاتری داد انتخاب بشه

**Reply by DoctorGuidance** [reply]
خود V2RayNG هم دکمه اپدیت داره، اول اینکه خودت سمت سرور تنظیم میکنی که معمولا هر 12 ساعت هست. 
دوم اینکه از تو برنامه هم میتونی بذاری مثلا 1 ساعت یک بار اپدیت شه:)

منظورت اینه پینگ کمتری داد!

**Comment by bensaeed** [comment]
V2Box 
هم هست

**Comment by M03ED** [comment]
برنامه V2box مشکل کم نداره ، مثلا یک موردی که همین امروز دیدم تو واتساپ پیا میومد و نمیرفت ، برای iOS 14 میتونید از Streisand استفاده کنید

**Comment by dexter19866** [comment]
v2rayAGN for android
https://play.google.com/store/apps/details?id=com.agn.v2ray&hl=fa

**Comment by mousade** [comment]
در اپدیت جدید مرزبان نام کاربر در برنامه هیدیفای نکست وارد نمیشود https://play.google.com/store/apps/details?id=app.hiddify.com&hl=en_US

**Reply by M03ED** [reply]
هیچ چیزی در این اپدیت برای این مورد تغییر نکرده ، مشکل از سمت مرزبان نیست

**Reply by mousade** [reply]
هیدیفای نکست به جای نام از subscription  استفاده میکنه جالبه وقتی لینک ساب مرزبان رو دیکود میکنی این میشه {"alg":"HS256","typ":"JWT"}{"sub":"Bahrain","access":"subscription","iat":1703784160}
توی تسخه ۰.۳ مرزبان  نام رو درست وارد میکنه

**Comment by qirmizialma** [comment]
Streisand (ios)

**Comment by goodboy34-tech** [comment]
Hiddify (all platform) with xray-core 🥰🥰

**Comment by Eslender73** [comment]
کسی لینک ساب ایمپورت foxray رو نمونشو داره؟ هرچیو میزنم کار نمیکنه

**Comment by jcltravels** [comment]
**Rocket Proxy** — iOS, iPadOS, macOS, Apple TV, Android

سلام. من سازنده‌ی این برنامه هستم، پس همین اول بگم که طرفِ بی‌طرفی نیستم.

لینک ساب مرزبان رو مستقیم ایمپورت می‌کنه (هم چند-لینکی که مرزبان برای هر نود می‌سازه)، و کانفیگ Clash/Stash YAML رو هم کامل می‌خونه — پس چیزی که الان برای Clash تولید می‌کنید بدون تغییر کار می‌کنه.

رایگانه، تبلیغ و تله‌متری نداره، و برای استفاده نیاز به ساخت حساب نیست.

- App Store (iPhone / iPad / Mac / Apple TV — یک لیست مشترک): https://apps.apple.com/app/id6785291194
- Google Play: https://play.google.com/store/apps/details?id=uk.co.jcltravels.rocketproxy
- راهنمای مخصوص مرزبان: https://github.com/jcltravels/RocketProxy/blob/main/distribution-kit/panels/marzban.md

نکته‌ای که شاید برای ادمین‌ها مهم باشه: دکمه‌ی ایمپورت فقط **ایمپورت** می‌کنه — خودکار وصل نمی‌شه و صفحه‌ی تأیید هم قابل رد کردن نیست. عمداً این‌طوره.

روی Apple TV ایمپورت یک‌کلیکی وجود نداره (tvOS مرورگر و دوربین نداره)؛ اونجا از «Receive from Nearby Device» استفاده می‌شه.

---

**In English, for anyone reading this thread later:**

I'm the developer of Rocket Proxy, so please weigh this accordingly.

It imports Marzban subscription links directly, including the one-link-per-node form Marzban emits, and it reads Clash/Stash YAML in full — proxies, policy groups, rules, rule providers, proxy providers and DNS — so whatever you already generate for Clash works unchanged. There is no new format to produce.

Free, no ads, no telemetry, no account required. One App Store listing covers iPhone, iPad, Mac and Apple TV; Android is on Play.

Two honest caveats, because admins always ask:

- The import button **only imports**. It never auto-connects, and the confirmation sheet is not skippable — that is deliberate.
- There is no one-click import on Apple TV. tvOS has no browser and no camera, so it uses *Receive from Nearby Device* instead.

Marzban setup guide: https://github.com/jcltravels/RocketProxy/blob/main/distribution-kit/panels/marzban.md

**Comment by rasel57** [comment]
Happ (Windows, Android, iOS, Android TV, Mac, all ...)
